README.md

```diff
@@ -36,6 +43,20 @@ This should open up the website in your browser in ``http://localhost:8080``
 
 The hot script will hot reload whenever you make changes in your files so that you can see your changes right away.
 
+## Docker Start
+
+Requirements for Windows OS
+- Windows 10 Pro
+- Docker
+- Hyper-V enabled
+
+1. Windows OS: Launch docker-compose
+```powershell
+launch_docker.ps1
```

Comment:
We need to make sure we have instructions/commands at the point of reading down and following the document line by line, that the user can run the app as expected up to this point.